src/Components/cart/cart.css

```diff
@@ -0,0 +1,55 @@
+.card-component {
+    margin-left: 2rem;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+  .card-margin {
+    margin: 2rem;
+  }
+  .card-horizontal {
+    display: flex;
+    flex-direction: row;
+    width: 55rem;
+    max-height: 31rem;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
```

Comment:
use var for color or make a box-shadow var so you can be consistent across the whole app.